launch/tier4_planning_launch/launch/planning.launch.xml

```diff
@@ -15,7 +15,7 @@
     <!-- mission planning module -->
     <group>
       <push-ros-namespace namespace="mission_planning"/>
-      <include file="$(find-pkg-share tier4_planning_launch)/launch/mission_planning/mission_planning.launch.py"/>
+      <include file="$(find-pkg-share tier4_planning_launch)/launch/mission_planning/mission_planning.launch.xml"/>
```

Comment:
The goal pose visualizer node only subscribes to the route topic. This topic is only published once per setting route, so I don't think it will affect performance much.